### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by Yulia-Smerechynska**

### CODE REVIEW

**Review by aquariusmaster — APPROVED** [review]

File: src/main/java/com/bringframework/BoboRegistry.java [hunk 1]
```diff
@@ -38,13 +40,16 @@ public BoboRegistry(Class<?>... itemClasses) {
...
-        if (candidates.size() == 0) {
+        int sizeOfCandidates = candidates.size();
+        if (sizeOfCandidates == 0) {
+            log.debug("There is no candidates for such BoboDefinition");
             throw new NoSuchBoboDefinitionException(
                     String.format(NO_SUCH_BOBO_DEFINIITON_ERROR, type.getSimpleName()));
         }
-        if (candidates.size() > 1) {
+        if (sizeOfCandidates > 1) {
+            log.error("Expected single manching bobo, but was {}", sizeOfCandidates);
```
Comment (DimaYakovenko) [inline src/main/java/com/bringframework/BoboRegistry.java:50]: typo in `manching`

File: src/main/java/com/bringframework/configurator/BoboConfiguratorScanner.java [hunk 2]
```diff
@@ -20,8 +23,11 @@
 
     private static BoboConfigurator createInstance(Class<? extends BoboConfigurator> configuratorClass) {
         try {
-            return configuratorClass.getDeclaredConstructor().newInstance();
+            BoboConfigurator boboConfigurator = configuratorClass.getDeclaredConstructor().newInstance();
+            log.debug("Create instance {} from declared constructor", boboConfigurator);
+            return boboConfigurator;
         } catch (Exception e) {
+            log.error("Can't create an instance of " + configuratorClass);
```
Comment (DimaYakovenko) [inline src/main/java/com/bringframework/configurator/BoboConfiguratorScanner.java:30]: pls use placeholder not concatenation

File: pom.xml [hunk 3]
```diff
@@ -58,6 +58,16 @@
             <version>${mockito-inline.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
```
Comment (DimaYakovenko) [inline pom.xml:63]: I guess lombok already includes this dependency

File: src/main/java/com/bringframework/configurator/BoboConfiguratorScanner.java (hunk 2 above)
Comment (Yulia-Smerechynska) [reply src/main/java/com/bringframework/configurator/BoboConfiguratorScanner.java:30]: fixed

File: src/main/java/com/bringframework/BoboRegistry.java (hunk 1 above)
Comment (Yulia-Smerechynska) [reply src/main/java/com/bringframework/BoboRegistry.java:50]: fixed

File: pom.xml (hunk 3 above)
Comment (Yulia-Smerechynska) [reply pom.xml:63]: deleted

**Review by DimaYakovenko — APPROVED** [review]

**Pull request merged**